app/src/main/java/com/byeboo/app/presentation/quest/aianswer/QuestAiAnswerViewModel.kt

```
@@ -1,29 +1,93 @@
+                        }
+                    }
+            }
         }
 
         fun onCloseClicked() {
-            // TODO: 나의 여정 퀘스트 메인 화면으로 이동
+            viewModelScope.launch {
+                if (aiAnswerEntryPointArg == AiAnswerEntryPoint.QUEST) {
+                    _sideEffect.emit(QuestAiAnswerSideEffect.NavigateToQuest)
+                } else {
+                }
```

detekt (1.23.8)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyElseBlock)